src/nih_wayfinding/test/spec/routing/directions-service.spec.js

```diff
@@ -305,6 +307,21 @@ describe('nih.routing: Directions', function () {
         httpBackend.verifyNoOutstandingRequest();
     });
 
+    it('should set OTP query parameters based on user preferences', function () {
+        var user = ProfileService.createBlankProfile();
+        user.username = 'test';
+        user.save();
+        ProfileService.setCurrentUser(user.username);
+        user.setPreference('restFrequency', 1);
+        user.setPreference('assistanceRequired', true);
+        user.setPreference('assistanceType', 'manual');
+        user.save();
+        var params = Directions.getRequestParams();
```

Comment:
Does getRequestParams exist as public only so that it can be tested?

In this case, its fine because it's done, but normally you would want to call Directions.get and inspect the url called for the proper parameters.